src/server/pyproject.toml

```
@@ -0,0 +1,42 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+packages = ['dcp_server']
+
+[tool.setuptools.dynamic]
+dependencies = {file = ["requirements.txt"]}
+
+[project]
+name = "data-centric-tool-server"
+version = "0.1"
+requires-python = ">=3.8"
+description = ""
+# license = {file = "LICENSE.txt"}
+keywords = []
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Operating System :: OS Independent",
+]
+readme = "README.md"
+dynamic = ["dependencies"]
+authors = [
+  {name="Christina Bukas", email="christina.bukas@helmholtz-muenchen.de"}
+]
+maintainers = [
+  {name="Christina Bukas", email="christina.bukas@helmholtz-muenchen.de"}
+]
```

The list of authors and maintainers are incomplete.
```suggestion
authors = [
  {name="Christina Bukas", email="christina.bukas@helmholtz-muenchen.de"},
  {name="Helena Pelin", email="helena.pelin@helmholtz-muenchen.de"}
]
maintainers = [
  {name="Christina Bukas", email="christina.bukas@helmholtz-muenchen.de"},
  {name="Helena Pelin", email="helena.pelin@helmholtz-muenchen.de"}
]
```